component/tasks.js

```diff
@@ -89,6 +89,10 @@ class Tasks {
     const workspace = this;
     fsUtility.readDataSource(workspace, cb);
   }
+  refreshMiddleware(cb) {
+    const workspace = this;
```

Comment:
storing `this` looks unnecessary since its not used in a callback or more than once